src/main/java/com/gooddata/executeafm/afm/PopMeasureDefinition.java

```diff
@@ -29,21 +29,31 @@
 
     private final ObjQualifier popAttribute;
 
+    private final Integer popOffset;
+
     /**
-     * Creates new definition from given measure identifier referencing another measure in {@link Afm} and given
-     * attribute qualifier (should qualify date attribute)
-     * @param measureIdentifier measure identifier
-     * @param popAttribute "period over period" date attribute
+     * Creates new definition from given measure identifier referencing another measure in {@link Afm} and given attribute qualifier (should qualify date
+     * attribute)
+     *
+     * @param measureIdentifier
+     *         measure identifier
+     * @param popAttribute
+     *         "period over period" date attribute
+     * @param popOffset
+     *         the number of periods the time window defined by {@code popAttribute} is offset by to the past (when value is negative) or future (when value
+     *         is positive)
      */
     public PopMeasureDefinition(@JsonProperty("measureIdentifier") final String measureIdentifier,
-                                @JsonProperty("popAttribute") final ObjQualifier popAttribute) {
+                                @JsonProperty("popAttribute") final ObjQualifier popAttribute,
+                                @JsonProperty("popOffset") final Integer popOffset) {
```

Comment:
This is not backward compatible change. Is this parameter mandatory or can we add constructor also without it?

Reply:
It is optional. I will add the constructor.